.circleci/config.yml

```diff
@@ -160,7 +160,7 @@ workflows:
             - go-fmt-and-vet
             - linters
       - build-distro:
-          OS: "darwin freebsd linux windows"
```

Comment:
why are we removing the darwin build?

Reply:
Sorry, didn't update PR comment.

* we don't release docker images for this arch
* breaks on golang 1.16 (https://golang.org/doc/go1.16#386)

Reply:
See https://app.circleci.com/pipelines/github/hashicorp/consul-k8s/2088/workflows/35dc4d70-b494-4ae7-9e65-2006c386ace7/jobs/11104

Reply:
Ah I see. Yep, and it looks like we [don't release](https://releases.hashicorp.com/consul-k8s/0.26.0-beta3/) darwin/386 binary either.